pwndbg/gdblib/arch.py

```diff
@@ -42,6 +42,32 @@
 }
 
 
+def read_thumb_bit() -> int | None:
+    """
+    Return 0 or 1, representing the status of the Thumb bit in the current Arm architecture
+
+    Return None if the Thumb bit is not relevent to the current architecture
+    """
+    if pwndbg.gdblib.arch.current == "arm":
+        # When program initially starts, cpsr may not be readable
+        cpsr = pwndbg.gdblib.regs.cpsr
+        if cpsr is not None:
```

Comment:
```suggestion
        if (cpsr := pwndbg.gdblib.regs.cpsr) is not None:
```

Since we support Ubuntu >= 20.04 and it uses Py >= 3.8, we can use this new and fancy assignment operator.